### PR TITLE
test: update for build features (Cypress)

### PR DESCRIPTION
**Pull request by DlgSHi**
### Additional context
- add missing test cases for build and buildClassic features;
- update common-steps.js file
- update cypress.env.json file
- update version of Cypress to 3.8.2 and Cucumber preprocessor to 2.0.1 (fix for Windows OS)

### CODE REVIEW

File: cypress/features/build/build.feature [hunk 1]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+      | component          | data-component     |
+      | dialog-full-screen | dialog-full-screen |
+      | dialog             | dialog             |
+      | flash              | flash              |
+      | pages              | page               |
+      | confirm            | confirm            |
 
   @build
   Scenario Outline: Component <component> without activation button
     When I open "<component>" component iframe
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: Please add 'in' before iframe. We should try to keep the convention using similar methods, e.g below you used `When I open "<component>" component page validations **in iframe**`
```suggestion
    When I open "<component>" component in iframe
```

File: cypress/features/build/build.feature [hunk 2]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+      | table-ajax                       | table-ajax           |
+      | table                            | table                |
+      | tabs                             | tabs                 |
+      | tile                             | tile                 |
+      | toast                            | toast                |
+      | tooltip                          | tooltip              |
+
+  @build
+  Scenario Outline: Component <component> default page and verify element
+    When I open "<component>" component iframe
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: Please add 'in' before iframe, as above
```suggestion
    When I open "<component>" component in iframe
```

File: cypress/features/build/build.feature [hunk 3]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+    Examples:
+      | component                     | data-element |
+      | experimental-groupedcharacter | input        |
+      | experimental-number-input     | input        |
+      | experimental-select           | input        |
+      | experimental-textbox          | input        |
+
+  @build
+  Scenario: Component experimental-textarea default page and verify element by Name
+    When I open "experimental-textarea" component iframe
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: Please add 'in' before iframe, as above
```suggestion
    When I open "experimental-textarea" component in iframe
```

File: cypress/features/build/build.feature [hunk 4]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+    Then "<data-element>" element is visible
+    Examples:
+      | component                     | data-element |
+      | experimental-groupedcharacter | input        |
+      | experimental-number-input     | input        |
+      | experimental-select           | input        |
+      | experimental-textbox          | input        |
+
+  @build
+  Scenario: Component experimental-textarea default page and verify element by Name
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: I think it should be
```suggestion
  Scenario: Verify element by Name in  component experimental-textarea default page
```

File: cypress/features/build/build.feature [hunk 5]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+      | experimental-switch              | Switch               |
+      | table-ajax                       | table-ajax           |
+      | table                            | table                |
+      | tabs                             | tabs                 |
+      | tile                             | tile                 |
+      | toast                            | toast                |
+      | tooltip                          | tooltip              |
+
+  @build
+  Scenario Outline: Component <component> default page and verify element
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: I think it should be
```suggestion
  Scenario Outline: Verify element in component <component> default page
```

File: cypress/features/build/build.feature [hunk 6]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+      | experimental-select           | input        |
+      | experimental-textbox          | input        |
+
+  @build
+  Scenario: Component experimental-textarea default page and verify element by Name
+    When I open "experimental-textarea" component iframe
+    Then "textarea" element is visible by name
+
+  @build
+  Scenario: Component experimental-textarea validations page and verify element by Name
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: I think it should be
```suggestion
  Scenario: Verify element by name in component experimental-textarea validations page
```

File: cypress/features/build/build.feature [hunk 7]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+    When I open "experimental-textarea" component page validations in iframe
+    Then "textarea" element is visible by name
+
+  @build
+  Scenario: Component experimental-textarea autoFocus page and verify element by Name
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: I think it should be
```suggestion
  Scenario: Verify element by Name in component experimental-textarea autoFocus page
```

File: cypress/features/build/build.feature [hunk 8]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+    When I open "<component>" component page autoFocus in iframe
+    Then "<data-component>" component is visible
+    Examples:
+      | component                  | data-component |
+      | experimental-checkbox      | checkbox       |
+      | experimental-date-input    | date           |
+      | experimental-decimal-input | decimal        |
+
+  @build
+  Scenario Outline: Component <component> autoFocus page and verify element
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: I think it should be
```suggestion
  Scenario Outline: Verify element in component <component> autoFocus page
```

File: cypress/features/build/build.feature [hunk 9]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+    When I open "<component>" component page autoFocus in iframe
+    Then "<data-element>" element is visible
+    Examples:
+      | component                     | data-element |
+      | experimental-groupedcharacter | input        |
+      | experimental-number-input     | input        |
+      | experimental-select           | input        |
+
+  @build
+  Scenario: Component experimental-textbox autoFocus multiple page and verify element
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: I think it should be
```suggestion
  Scenario: Verify element in component experimental-textbox autoFocus multiple page
```

File: cypress/features/build/build.feature [hunk 10]
```diff
@@ -22,73 +22,162 @@ Feature: Build tests
...
+    When I open "experimental-textbox" component page autoFocus multiple in iframe
+    Then "input" element is visible
+
+  @build
+  Scenario Outline: Component <component> multiple page and verify element
```
Comment (KatarzynaQA) [inline cypress/features/build/build.feature]: I think it should be 
```suggestion
  Scenario Outline: Verify element in component <component> multiple page
```

File: cypress/features/build/buildClassic.feature [hunk 11]
```diff
@@ -22,81 +22,144 @@ Feature: Build tests for classic stories
...
+      | step-sequence                    | step-sequence        |
+      | experimental-switch              | Switch               |
+      | table-ajax                       | table-ajax           |
+      | table                            | table                |
+      | tabs                             | tabs                 |
+      | toast                            | toast                |
+      | tooltip                          | tooltip              |
+
+  @build
+  Scenario Outline: Component <component> default page and verify element
```
Comment (KatarzynaQA) [inline cypress/features/build/buildClassic.feature]: I think it should be
```suggestion
  Scenario Outline: Verify element in component <component> default page
```

File: cypress/features/build/buildClassic.feature [hunk 12]
```diff
@@ -22,81 +22,144 @@ Feature: Build tests for classic stories
...
+    Then "<data-element>" element is visible
+    Examples:
+      | component                     | data-element |
+      | experimental-groupedcharacter | input        |
+      | experimental-number-input     | input        |
+      | experimental-select           | input        |
+      | experimental-textbox          | input        |
+
+  @build
+  Scenario: Component experimental-textarea default page and verify element by Name
```
Comment (KatarzynaQA) [inline cypress/features/build/buildClassic.feature]: I think it should be
```suggestion
  Scenario:  Verify element by Name in component experimental-textarea default page
```

File: cypress/features/build/buildClassic.feature [hunk 13]
```diff
@@ -22,81 +22,144 @@ Feature: Build tests for classic stories
...
+    When I open "experimental-textarea" component for classic story in iframe
+    Then "textarea" element is visible by name
+
+  @build
+  Scenario: Component experimental-textarea validations page and verify element by Name
```
Comment (KatarzynaQA) [inline cypress/features/build/buildClassic.feature]: I think it should be 
```suggestion
  Scenario:  Verify element by Name in component experimental-textarea validations page
```

File: cypress/features/build/buildClassic.feature [hunk 14]
```diff
@@ -22,81 +22,144 @@ Feature: Build tests for classic stories
...
+    When I open "Validations" component buttonToogleGroup classic page validation in iframe
+    Then "button-toggle-group" component is visible
+
+  @build
+  Scenario: Component Loader legacy spinner in iframe default page
+    When I open "loader" component legacy spinner classic page in iframe
+    Then "spinner" component is visible
 
   @build
   Scenario Outline: Deprecated component <component> iframe
```
Comment (KatarzynaQA) [inline cypress/features/build/buildClassic.feature]: Please add 'in'
```suggestion
  Scenario Outline: Deprecated component <component> in iframe
```

File: cypress/features/build/buildClassic.feature [hunk 15]
```diff
@@ -22,81 +22,144 @@ Feature: Build tests for classic stories
...
     When I open deprecated "<component>" component iframe
```
Comment (KatarzynaQA) [inline cypress/features/build/buildClassic.feature]: Please add 'in'
```suggestion
    When I open deprecated "<component>" component in iframe
```

File: cypress/features/build/build.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/build.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/build.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/build.feature (hunk 4 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/build.feature (hunk 5 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/build.feature (hunk 6 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/build.feature (hunk 7 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/build.feature (hunk 8 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/build.feature (hunk 9 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/build.feature (hunk 10 above)
Comment (DlgSHi) [reply cypress/features/build/build.feature]: fixed

File: cypress/features/build/buildClassic.feature (hunk 11 above)
Comment (DlgSHi) [reply cypress/features/build/buildClassic.feature]: fixed

File: cypress/features/build/buildClassic.feature (hunk 12 above)
Comment (DlgSHi) [reply cypress/features/build/buildClassic.feature]: fixed

File: cypress/features/build/buildClassic.feature (hunk 13 above)
Comment (DlgSHi) [reply cypress/features/build/buildClassic.feature]: fixed

File: cypress/features/build/buildClassic.feature (hunk 14 above)
Comment (DlgSHi) [reply cypress/features/build/buildClassic.feature]: fixed

File: cypress/features/build/buildClassic.feature (hunk 15 above)
Comment (DlgSHi) [reply cypress/features/build/buildClassic.feature]: fixed

**Pull request merged**